deploy/configs/libraries/modules.yaml

```diff
@@ -50,6 +50,7 @@ modules:
       - '%gcc'
       - '%intel'
       - '%clang'
+      - '%nvhpc'
```

Comment:
Shouldn't we add this to all module configs?

Reply:
Now done (I think) in 6e39171a083817b5d3f647e3098a10168f9c8661.